### DISCUSSION TITLE
请问5.X版本现在不能实现像4.X版本中的这种玫瑰分面图了吗

### DISCUSSION THREAD

**Discussion by Can-Xu** [antvis/G2#5096] [Q&A] [answered]
![1684996762228](https://github.com/antvis/G2/assets/37643989/08112441-5492-41c5-96b4-27d7f46eb54f)

**Comment by pearmini** [comment, marked as answer]
可以参考下面两个例子：

- https://g2.antv.antgroup.com/examples/composition/facet/#rect-row
- https://g2.antv.antgroup.com/zh/examples/composition/facet/#rect-pie
